<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <widget class="QPushButton" name="loginBtn">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>137</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登陆</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="accountTab">
    <attribute name="title">
     <string>账号登陆</string>
    </attribute>
    <widget class="QLineEdit" name="accountEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="placeholderText">
      <string>请输入账号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="placeholderText">
      <string>请输入密码</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="verificationTab">
    <attribute name="title">
     <string>短信登陆</string>
    </attribute>
    <widget class="QLineEdit" name="phoneNumEdit">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>8</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>11</number>
     </property>
     <property name="placeholderText">
      <string>请 输入手机账号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="verifyEdit">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>38</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
     <property name="placeholderText">
      <string>请输入短信验证码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="verifyBtn">
     <property name="geometry">
      <rect>
       <x>127</x>
       <y>34</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>短信验证</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="tipLab">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>120</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
